OpenRA.Mods.Common/Traits/World/OrderEffects.cs

```diff
@@ -0,0 +1,69 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Mods.Common.Effects;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+    [Desc("Renders an effect at the order target locations.")]
+    public class OrderEffectsInfo : TraitInfo
+    {
+        [Desc("The image to use.")]
+        [FieldLoader.Require]
+        public readonly string FlashImage;
```

Comment:
Can we please name these `TerrainFlash(Image|Sequence|Palette)`?

I plan to file a followup PR that adds an `ActorFlashEffect` option, so it would be good if these could be clear about being related to the terrain from the start.

Reply:
LGTM and will merge after this change.